runbot_build_instructions/runbot_repo.py

```diff
@@ -53,6 +53,10 @@ class RunbotRepo(models.Model):
         'Custom Pre-build Command',
         help=ARGUMENTS_HELP,
     )
+    custom_odoo_path = fields.Char(
+        'Custom Odoo Path',
+        help="Relative path to Odoo",
```

Comment:
could you make clear here if this is the path to the odoo directory (the path the contains `openerp-server.py`), what I assume, or the path to the odoo executable?